app/controllers/event_actions_controller.rb

```diff
@@ -72,6 +72,6 @@ def set_event_action
   def event_action_params
     params
       .require(:event_action)
-      .permit(:event_receiver_id, :job_type_id, :email_address, :webhook_url, :webhook_body, :type)
+      .permit(:event_receiver_id, *EventAction::PROPERTIES)
```

Comment:
Good call with the splat operator, and reusing it in the `event_receivers_controller` as well. 👍